libs/shared/src/postgresdb.module.ts

```diff
@@ -0,0 +1,24 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot({
+      isGlobal: true,
+      envFilePath: './.env',
+    }),
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule],
+      useFactory: (configService: ConfigService) => ({
+        type: 'postgres',
+        url: `postgresql://${configService.get('POSTGRES_USER')}:${configService.get('POSTGRES_PASSWORD')}@${configService.get('POSTGRES_HOST')}:${configService.get('POSTGRES_PORT')}/${configService.get('POSTGRES_DB')}`,
+        autoLoadEntities: true,
+        synchronize: true, // TODO: remove this in production
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Remove synchronize option in production</b></div>
<div id="fix">

Consider removing the `synchronize: true` option in production environments. This option automatically alters the database schema to match your entities, which can lead to data loss in production.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -13,7 +13,7 @@
        useFactory: (configService: ConfigService) => ({
          type: 'postgres',
          url: `postgresql://${configService.get('POSTGRES_USER')}:${configService.get('POSTGRES_PASSWORD')}@${configService.get('POSTGRES_HOST')}:${configService.get('POSTGRES_PORT')}/${configService.get('POSTGRES_DB')}`,
          autoLoadEntities: true,
 -        synchronize: true, // TODO: remove this in production
 +        synchronize: configService.get('NODE_ENV') !== 'production',
        }),
 
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/messenger-api/pull/6#issuecomment-2448418554>#48ef5b</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged